src/pal/inc/pal.h

```diff
@@ -4315,7 +4316,7 @@ See MSDN doc for memcpy
 EXTERN_C
 PALIMPORT
 DLLEXPORT
-void *PAL_memcpy (void *dest, const void *src, size_t count);
+PVOID PAL_memcpy (PVOID dest, const void * src, size_t count);
```

Comment:
Undo

Reply:
Ah, i thought all PAL_* are not considered (and tested) as CRT functions?
CRT: `memcpy` and `memcpy_s` 
non-CRT: `PAL_memcpy`

Reply:
`PAL_memcpy` is a Windows-compatible shim over regular `memcpy` that is then used to override memcpy via `define`. It is more a CRT function than not.